spec/spec_helper.rb

```diff
@@ -1 +1,2 @@
+require 'rubygems'
```

Comment:
It is not required. Check out bundler.

`gem install bundler`

and in the ssh directory type `bundle install`

and put `bundle exec` before the rspec commands, like

`bundle exec rake spec_standalone`